src/Umbraco.Web.UI.Client/src/views/propertyeditors/listview/listview.html

```diff
@@ -11,35 +11,85 @@
+                               <localize key="actions_create">Create</localize> {{listViewAllowedTypes[0].name}}
+                           </span>
                            <span class="caret"></span>
                        </a>
+
                        <ul class="dropdown-menu">
-                           <li ng-repeat="contentType in listViewAllowedTypes | orderBy:'name':false">
-                               <a href="#/{{entityType}}/{{entityType}}/edit/{{contentId}}?doctype={{contentType.alias}}&create=true">
-                                   <i class="icon-{{contentType.cssClass}}"></i>
-                                   {{contentType.name}}
+                           <li ng-repeat="(key, value) in listViewAllowedTypes[0].blueprints">
+                               <a ng-click="createFromBlueprint(entityType, listViewAllowedTypes[0].blueprints.alias, key)">
+                                   <i class="{{listViewAllowedTypes[0].icon}}"></i>
+                                   {{value}}
                                </a>
                            </li>
                        </ul>
                    </div>
-                   <div class="btn-group" ng-show="listViewAllowedTypes.length === 1">
-                       <a class="btn" href="#/{{entityType}}/{{entityType}}/edit/{{contentId}}?doctype={{listViewAllowedTypes[0].alias}}&create=true">
-                           <i class="icon-{{listViewAllowedTypes[0].cssClass}}"></i>
-                           <localize key="actions_create">Create</localize> {{ listViewAllowedTypes[0].name }}
+
+                   <div class="btn-group" ng-show="createAllowedButtonMultiWithBlueprints">
+                       <a class="btn dropdown-toggle" data-toggle="dropdown" href="#">
+                           <localize key="actions_create">Create</localize>
+                           <span class="caret"></span>
                        </a>
+                       <ul class="dropdown-menu">
+                           <li ng-repeat="contentType in listViewAllowedTypes | orderBy:'name':false">
+                               <a ng-click="createBlank(entityType,contentType.alias)">
+                                   <i class="{{contentType.icon}}"></i>
+                                   {{contentType.name}} <span ng-show="contentType.blueprints && contentType.blueprints.length != 0" style="text-transform: lowercase;">(<localize key="blueprints_blankBlueprint">blank</localize>)</span>
+                               </a>
+                               <a ng-repeat="(key, value) in contentType.blueprints" ng-click="createFromBlueprint(entityType,contentType.alias , key)">
+                                   &nbsp;&nbsp;<i class="{{contentType.icon}}"></i>
+                                   {{value}}
+                               </a>
+                           </li>
+                       </ul>
                    </div>
+
+                   <ul class="umb-actions umb-actions-child">

```

Comment:
I'm unsure what this block of code is doing? As far as I can tell there is no `$scope.docType` object so I don't think this entire html block is actually used, can you clarify @KimHolzmann  ?